tests/src/test/scala/pathy/path.scala

```diff
@@ -100,133 +84,194 @@ class PathSpecs extends Specification with ScalaCheck {
     ).toList must_== List("c", "foo", "bar", "flat.md")
   }
 
-  "parseRelFile - image.png" in {
-    parseRelFile("image.png") must beSome(file("image.png"))
-  }
+  "parseRelFile" >> {
+    "should successfully parse" >> {
 
-  "parseRelFile - ./image.png" in {
-    parseRelFile("./image.png") must beSome(file("image.png"))
-  }
+      """simple file name with extension such as "image.png"""" in {
+        parseRelFile("image.png") must beSome(file("image.png"))
+      }
 
-  "parseRelFile - foo/image.png" in {
-    parseRelFile("foo/image.png") must beSome(dir("foo") </> file("image.png"))
-  }
+      """preceeded with current directory such as "./image.png"""" in {
+        parseRelFile("./image.png") must beSome(file("image.png"))
+      }
 
-  "parseRelFile - ../foo/image.png" in {
-    parseRelFile("../foo/image.png") must beSome(currentDir <::> dir("foo") </> file("image.png"))
-  }
+      """path with two segments such as "foo/image.png"""" in {
+        parseRelFile("foo/image.png") must beSome(dir("foo") </> file("image.png"))
+      }
 
-  "parseRelFile - /foo/image.png" in {
-    parseRelFile("/foo/image.png") must beNone
-  }
+      """preceeded by a parent directory reference such as "../foo/image.png"""" in {
+        parseRelFile("../foo/image.png") must beSome(currentDir <::> dir("foo") </> file("image.png"))
+      }
+    }
 
-  "parseRelFile - foo/" in {
-    parseRelFile("foo/") must beNone
-  }
+    "should fail to parse" >> {
 
-  "parseAbsFile - /image.png" in {
-    parseAbsFile("/image.png") must beSome(rootDir </> file("image.png"))
-  }
+      """with a leading / such as "/foo/image.png"""" in {
+        parseRelFile("/foo/image.png") must beNone
+      }
 
-  "parseAbsFile - /foo/image.png" in {
-    parseAbsFile("/foo/image.png") must beSome(rootDir </> dir("foo") </> file("image.png"))
-  }
+      "with a trailing /" in {
```

Comment:
Because it has no "example"? Haha, okay fine, I don't feel too strongly, so if you really prefer it this way, I can do that. This description is pretty simple though, sometimes it requires more cognitive effort to craft an example in one's head of what the description means.